docs/FeatureManagement/TimeWindowFilter.v2.0.0.schema.json

```diff
@@ -0,0 +1,160 @@
+      "enum": [
+        "Microsoft.TimeWindow"
+      ]
+    },
+    "parameters": {
+      "$id": "#/properties/parameters",
+      "type": "object",
+      "title": "Time Window Filter Parameters",
+      "properties": {
+        "Start": {
+          "$id": "#/properties/parameter/properties/Start",
+          "type": "string",
+          "title": "Time Window Filter Start Parameter",
+          "examples": [
+            "Wed, 1 May 2024 12:00:00 GMT",
+            "Wed, 1 May 2024 20:00:00 +0800"
+          ]
+        },
+        "End": {
+          "$id": "#/properties/parameters/properties/End",
+          "type": "string",
+          "title": "Time Window Filter End Parameter",
+          "examples": [
+            "Thu, 2 May 2024 12:00:00 GMT",
+            "Thu, 2 May 2024 20:00:00 +0800"
+          ]
+        },
+        "Recurrence": {
+          "$id": "#/properties/parameters/properties/Recurrence",
+          "type": "object",
+          "title": "Time Window Filter Recurrence Parameter",
+          "properties": {
+            "Pattern": {
+              "$id": "#/properties/parameters/properties/Recurrence/properties/Pattern",
+              "type": "object",
+              "title": "Time Window Filter Recurrence Pattern Parameter",
+              "required": [
+                "Type"
+              ],
+              "properties": {
+                "Type": {
+                  "$id": "#/properties/parameters/properties/Recurrence/properties/Pattern/properties/Type",
+                  "type": "string",
+                  "title": "Time Window Filter Recurrence Pattern Type Parameter",
+                  "enum": [
+                    "Daily",
+                    "Weekly"
+                  ]
+                },
+                "Interval": {
+                  "$id": "#/properties/parameters/properties/Recurrence/properties/Pattern/properties/Interval",
+                  "type": "number",
+                  "title": "Time Window Filter Recurrence Pattern Interval Parameter",
+                  "default": 1
+                },
+                "DaysOfWeek": {
+                  "$id": "#/properties/parameters/properties/Recurrence/properties/Pattern/properties/DaysOfWeek",
+                  "type": "array",
+                  "title": "Time Window Filter Recurrence Weekly Pattern DaysOfWeek Parameter",
+                  "items": {
+                    "$id": "#/properties/parameters/properties/Recurrence/properties/Pattern/properties/DaysOfWeek/items",
+                    "type": "string",
+                    "title": "DayOfWeek",
+                    "enum": [
+                      "Monday",
+                      "Tuesday",
+                      "Wednesday",
+                      "Thursday",
+                      "Friday",
+                      "Saturday",
+                      "Sunday"
+                    ]
+                  }
+                },
+                "FirstDayOfWeek": {
```

Comment:
Just want to confirm, we didn't provide "Sunday" as default value?

Reply:
Yes. I forgot to specify the default value. Added.